src/main/java/org/folio/edge/oaipmh/Verb.java

```diff
@@ -121,6 +123,7 @@ private void validateIllegalParams(RoutingContext ctx, List<OAIPMHerrorType> err
     ctx.request().params().entries().stream()
       .map(Entry::getKey)
       .filter(param -> !allParams.contains(param))
+      .filter(param -> !excludeParams.contains(param))
```

Comment:
The `excludeParams` are being added to `allParams` in `Verb` constructor:
```     allParams = concat(concat(requiredParams.stream(), optionalParams.stream()), excludeParams.stream())
.collect(toSet());
```
So I think this extra check is not required.

Reply:
Good call.  At one point I had changes in here to support a case-insensitive apiKey parameter, and needed this here because I was lower-casing the query arg key and checking against the excluded params.  I later ripped that functionality out and neglected to realized this was no longer needed.